bitnami/redmine/Chart.yaml

```
@@ -23,3 +23,17 @@ maintainers:
 engine: gotpl
 annotations:
   category: ProjectManagement
+dependencies:
+  - name: common
+    version: 0.x.x
```

```suggestion
    version: 1.x.x
```
the bitnami/common subchart has been updated to a new major in order to reflect the new `apiVersion: v2`

---

👍